### PR TITLE
CVSB-5385 Retro certificates will also be issued when test-results is sent with an empty body

### PR DESCRIPTION
**Pull request by amarasescu**

### CODE REVIEW

File: tests/unit/reportGen.unitTest.ts [hunk 1]
```diff
@@ -106,18 +107,19 @@ describe("report-gen", () => {
                     });
                 });
 
-                context("and the response is malformed", () => {
-                    it("should throw an error", () => {
-                        LambdaMockService.changeResponse("cvs-svc-test-results", "tests/resources/test-results-malformed-response.json");
+                context("and the response is 200", () => {
```
Comment (KHodderDVSA) [inline tests/unit/reportGen.unitTest.ts]: Need test for 404 response handling in LambdaService too
